### ISSUE TITLE
Advanced examples with multiple components?

### ISSUE THREAD

**Issue by soullivaneuh** [sveltejs/component-template#18]
I just follow the README and setup your template.

Now I'm facing a blank `index.svelte` file.

Having some documentation and example and how to export multiple templates would be welcomed! :+1:

**Comment by soullivaneuh** [comment]
I found an example here : https://github.com/c0bra/svelma/blob/master/src/index.js

But it is not a svelte file. What is the difference comparing this template?

**Comment by rixo** [comment]
Currently, Svelma is shipping precompiled components. That means you can use them anywhere, especially outside of a Svelte application. However, that also means that they are bundled with their own copy of Svelte's runtime.

This is suboptimal if you use the components in a Svelte application that you compile yourself. And, depending on the Svelte features that are used by your components, mixing different Svelte runtime can also cause weird issues for the consumer. That's why you've got this recommendation in the README of this template:

> Your package.json has a "svelte" field pointing to src/index.svelte, which allows Svelte apps to import the source code directly

I've not tested this extensively, but apparently the following seems to work (at least with `rollup-plugin-svelte`) for what you want to do.

In your `package.json`:

~~~diff
-  "svelte": "src/index.svelte",
+  "svelte": "src/index.js",
~~~

In `index.js`:

~~~js
export { default as FirstComponent } from './FirstComponent.svelte'
export { default as SecondComponent } from './SecondComponent.svelte'
// etc...
~~~

**Comment by soullivaneuh** [comment]
Thanks for your knowledge share @rixo ! :+1: 

I'll try it as soon as possible.

**Comment by mvolkmann** [comment]
I have documented all the steps required to get this to work at https://github.com/mvolkmann/svelte-components. Some of these steps could be removed with minor changes to component-template.

**Comment by mvolkmann** [comment]
Which is the more common case, that people want to use this to create a library containing one component or multiple? My guess is that multiple is more common. But this is set up to only support one and changes are required to support multiple. I think it should be changed to support multiple components out of the box.

**Comment by Conduitry** [comment]
Closed this in #20 which makes it a bit easier to transition to a multiple-component package and adds a short note to the readme.

**Comment by soullivaneuh** [comment]
Thanks a lot @Conduitry, great addition to simplify the usage and motivate people to do more awesome stuff with svelte! :+1: 

I'll try it right now.
